api/lib/notifications/notificationController.js

```diff
@@ -62,23 +77,32 @@ class NotificationController {
     uid,
     addressObjects,
     notificationType,
-    dataSources
+    dataSources,
+    pushToken
   ) {
     let topics = []
     if (notificationType === 'email') {
       topics = (
         await getDefaultEMailSubscriptions(addressObjects, dataSources)
```

Comment:
```suggestion
        await getDefaultEmailSubscriptions(addressObjects, dataSources)
```

Reply:
need to change this in more then one place?

Reply:
Done